extensions/wikia/PhalanxII/js/SpecialPhalanx.js

```diff
@@ -30,34 +32,34 @@ require(['jquery', 'mw', 'phalanx', 'BannerNotification'], function($, mw, phala
 
 		// handle "bulk mode" button
 		on('click', '#enterbulk', function(ev) {
-			var singleModeWrapper = $('#singlemode'),
-				bulkModeWrapper = $('#bulkmode');
-
 			// SUS-1191: preload single filter contents into bulk mode textbox
-			$bulkFilter.val($filter.val());
+			bulkFilter.val(filter.val());
 
 			singleModeWrapper.slideUp();
 			bulkModeWrapper.slideDown();
 
+			// hide validation message when switching modes
+			$('#validateMessage').hide();
+			// clear input field when switching modes, for validation purposes
+			filter.val('');
 			// SUS-1191: autofocus newly revealed input field
-			$bulkFilter.focus();
+			bulkFilter.focus();
 		}).
 
 		// handle "single mode" button
 		on('click', '#entersingle', function(ev) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "ev". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)